### ISSUE TITLE
🚧Improve release notes grammar and spelling

### ISSUE THREAD

**Issue by CalvinWilkinson** [KinsonDigital/Velaptor#394]
### Complete The Item Below

- [X] I have updated the title without removing the 🚧 emoji.

### Description

Go through all of the preview release notes and use the Grammarly extension to improve all of the preview release notes.
Once this is done, each GitHub release should have the release notes updated.

### Acceptance Criteria

- [x] All grammar and spelling errors fixed in preview release notes
- [x] All releases that have release note changes are updated.

### ToDo Items

- [X] Change type labels added to this issue.  Refer to the _**Change Type Labels**_ section below.
- [X] Priority label added to this issue.  Refer to the _**Priority Type Labels**_ section below.
- [X] Issue linked to the correct project _(if applicable)_.
- [x] Issue linked to the correct milestone _(if applicable)_.
- [x] Draft pull request created and linked to this issue _(only required with code changes)_.

### Additional Information:

**_<details closed><summary>Change Type Labels</summary>_**

| Change Type         | Label                |
|---------------------|----------------------|
| Bug Fixes           | `🐛bug` |
| Breaking Changes    | `🧨breaking changes` |
| New Feature         | `✨new feature` |
| Workflow Changes    | `workflow` |
| Code Doc Changes    | `🗒️documentation/code` |
| Product Doc Changes | `📝documentation/product` |
</details>

**_<details closed><summary>Priority Type Labels</summary>_**

| Priority Type       | Label             |
|---------------------|-------------------|
| Low Priority        | `low priority`    |
| Medium Priority     | `medium priority` |
| High Priority       | `high priority`   |
</details>

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct.

**Comment by kselena** [comment]
@CalvinWilkinson Do you need me to pick this up?

**Comment by CalvinWilkinson** [comment]
> @CalvinWilkinson Do you need me to pick this up?

Yes please

**Comment by kselena** [comment]
@CalvinWilkinson is this issue for the template?

**Comment by CalvinWilkinson** [comment]
> @CalvinWilkinson is this issue for the template?

No.  This is simply to make sure that the old preview release notes are good to go is all
